lumicks/pylake/calibration_item.py

```diff
@@ -0,0 +1,227 @@
+import re
```

Comment:
I don't like this module at the top level personally? It feels more like an implementation detail. could we move it into `details` maybe?

Reply:
Yeah, good point to not clutter up the main folder, but as we discussed, probably move it to `force_calibration` rather than `details` (considering it is user facing).